### ISSUE TITLE
Balancing/smoothening of lategame content (pepper2000 material)

### ISSUE THREAD

**Issue by dzfischer** [caveman2cosmos/Caveman2Cosmos#696]
For examples, see:
https://forums.civfanatics.com/threads/caveman-2-cosmos-ideas-discussions-thread.377892/page-1011#post-15936809

**Comment by raxo2222** [comment]
Well ALL buildings need complete overhaul, just like current unit review project.

**Comment by dzfischer** [comment]
lol true, but seems like these are in a rougher state, comparatively. Mostly just didn't want the feedback to disappear 😅

**Comment by Thunderbrd** [comment]
> Well ALL buildings need complete overhaul, just like current unit review project.

Agree, along with a number of new tags and processes that address numerous issues we've collected through the years.

But in the meantime the later era stuff indeed has less refinement than the earlier stuff so work on those things doesn't hurt I suppose.

**Comment by dzfischer** [comment]
@Thunderbrd do you have a collection of the new tag ideas, processes, etc you have in mind? If not, might be good to have it written down somewhere on git at some point 😄
Doesn't have to be inclusive at inception, can always add to it ofc.

**Comment by MattOttawa** [comment]
I wonder if Faustmouse would be interested in making prs?

**Comment by Thunderbrd** [comment]
hmm... I did make some posts on it on the forum a while back, probably just a bit before we started working with Git.  I'll have to hunt that down.  In essence, the main gist was highly granular tags for yield and commerce per population for access to a resource (and possibly even allowing for multiple resources to count as well so PER resource of a type).  This would be used for markets and such to make the final providing spot in the supply chain capable of giving a highly scaling base amount of yield/commerce for having access to what it sells to the consumer.  Then the types of buildings that GENERATE resources would tend to be more of a support drain to get them to function.  A lot of that would play into the zoning stuff below.

Then there was some stuff about a zoning project as well... I could work that out as a separate thing but it would be easiest to point someone good at translating it all into a git project writeup from the thread I was introducing it in.  I'll try to look for that later.  The base concept is that all cities get a base # of a zone type + 1 for each plot worked of a particular type of improvement and each building requires a zone type available to be constructed.  

There's also some plans with specialists and great people and constricting access to specialist types heavily, while citizens (basic laborers) even become something that a city can only sustain so many of depending on its buildings, and basically if a city overgrows its ability to employ citizens the base citizen type becomes an unemployed type that's basically the same as an angry citizen because they CAN'T find work (rather than simply refusing to because they don't like how things are going).

GP points would also come from improvements worked, such as Great Farmers actually being a thing you'd get access to by working farms and farm buildings (which would require farm zones available by having worked farms in the city radius as well).  You'd have minor trainable types of citizens like Farmers but they'd have limited and perhaps developable capacities to place resources while the Great Versions would be pretty much miraculously automatic at it. (As well as being able to do a lot of other normally GP type things they can't currently do.)  You'd have Great Foresters, Great Seamen, Great Prospectors, and probably more.

**Comment by Thunderbrd** [comment]
This was the post I was thinking of but I still don't think it's quite the overview I was looking for:
https://forums.civfanatics.com/threads/research-request-s.627473/page-3#post-15011566
